### PR TITLE
tests: add tf linting R checks & fix errors

### PR DESCRIPTION
**Pull request by katbyte**

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -61,17 +68,17 @@ testacc: fmtcheck
 debugacc: fmtcheck
 	TF_ACC=1 dlv test $(TEST) --headless --listen=:2345 --api-version=2 -- -test.v $(TESTARGS)
 
+website-lint:
+	@echo "==> Checking website against linters..."
+	@misspell -error -source=text -i hdinsight website/
+
```
Comment (tombuildsstuff) [inline GNUmakefile]: alphabetically this one wants to be below the other one?

File: azurerm/resource_arm_function_app.go [hunk 2]
```diff
@@ -205,7 +205,6 @@ func resourceArmFunctionApp() *schema.Resource {
 						"virtual_network_name": {
 							Type:     schema.TypeString,
 							Optional: true,
-							Computed: false,
```
Comment (tombuildsstuff) [inline azurerm/resource_arm_function_app.go]: is this intentional?

Comment (katbyte) [reply azurerm/resource_arm_function_app.go]: Yea, `Computed: false,` doesn't actually mean anything right?

File: GNUmakefile (hunk 1 above)
Comment (katbyte) [reply GNUmakefile]: yestrue but aesthetically and functionally I think it's better above given the last two massive blocks run together and i spent a while skimming the file looking for it

**Pull request merged**